main/src/com/google/refine/expr/functions/strings/Reinterpret.java

```diff
@@ -89,20 +89,20 @@ private Object reinterpret(String str, String decoder, String encoder) {
                 result = new String(bytes, encoder);
             }
         } catch (UnsupportedEncodingException e) {
-            return new EvalError(ControlFunctionRegistry.getFunctionName(this) + ": encoding '" + encoder + "' is not available or recognized.");
+            return new EvalError(ControlFunctionRegistry.getFunctionName(this) + ": target encoding '" + encoder + "' is not available or recognized.");
         }
                         
         return result;
     }
     
     @Override
     public String getDescription() {
-        return "Returns s reinterpreted thru the given encoder.";
+        return "Returns s reinterpreted thru the given target encoder and optional source decoder.";
```

Comment:
And this is a description of the function, which indeed describes well what it actually does, so keep it worded as this PR, or do we tweak to make friendly as well?

Reply:
OK sounds like the consensus is to phrase it this way as well, since this is user facing in the Help panel of Expression Editor:
```suggestion
        return "Returns s reinterpreted using a target encoding and optional source encoding.";
```